Client/Frontend/Menu/BookmarksViewController.swift

```diff
@@ -180,6 +180,12 @@ class BookmarksViewController: SiteTableViewController {
       print(error.description)
     }
     
+    if tableView.isEditing == false && bookmarksFRC?.fetchedObjects?.count == 0 {
```

Comment:
If table is blocked from being in edit mode while then this check isn't necessary.

Reply:
This was for checking when you open menu button. I will move it viewDidLoad() for clarity.